web/frontend/styles/casebook.scss

```diff
@@ -540,8 +539,9 @@ form.edit_content_resource, form.edit_content_section, form.edit_content_caseboo
           background-size: auto;
           &.action-delete {
             background-image: url('~static/images/ui/edit/elide.png');
-            height: 36px;
-            width: 36px;
+            height: 20px;
+            width: 20px;
```

Comment:
Curious about this height change as well

Reply:
Yeah, same as with the SVG: this was to make the delete button align with the little boxes that say "Text" or "Case", and not affect the height of the entry.